.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -132,20 +132,6 @@ fun BaseGradleBuildType.verifyTestFilesCleanupStep(daemon: Boolean = true) {
     }
 }
 
-private
-fun BaseGradleBuildType.tagBuildStep(model: CIBuildModel, daemon: Boolean = true) {
```

Comment:
Do you know what will happen on `release`? I suppose we will continue using the build step on `release`, since we are trying to use the TC configuration from the branch.

Reply:
I'm sure each branch uses their own configurations for build steps.